week1/calculus.ipynb

```diff
@@ -0,0 +1,1060 @@
+{
```

Comment:
<p>Also, mention that the chain rule is used in the Backpropagation step in Neural network to see the rate of change on some weight keeping all other weights/bias constants. This is the important step through which we can find rate of change in each neuron and helps to tune parameters for learning purposes.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/toggle-corp/ml-internship/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>